src/Authoring/WinRT.SourceGenerator/AotOptimizer.cs

```diff
@@ -90,7 +90,7 @@ private static string ToVtableLookupString(ISymbol symbol)
 
         internal static VtableAttribute GetVtableAttributeToAdd(ITypeSymbol symbol, Func<ISymbol, bool> isWinRTType, IAssemblySymbol assemblySymbol, bool isAuthoring, string authoringDefaultInterface = "")
         {
-            if (GeneratorHelper.HasNonInstantiatedWinRTGeneric(symbol) || GeneratorHelper.HasPrivateclass(symbol))

```

Comment:
Just curious: how come we removed that other condition?

Reply:
The nested type generation fixes the private class issue as seen in program.cs.